README.md

```diff
@@ -84,9 +110,10 @@ example.com {
 ```
 
 ## Building it
-Build from caddy repository and import  **caddy-run** plugin on file https://github.com/mholt/caddy/blob/master/caddy/caddymain/run.go :
+Build from caddy repository and import  **caddy-supervisor** plugin on file https://github.com/mholt/caddy/blob/master/caddy/caddymain/run.go :
 ```
 import (
-  _ "github.com/lucaslorentz/caddy-run/plugin"
+  _ "github.com/lucaslorentz/caddy-supervisor/httpplugin"
+  _ "github.com/lucaslorentz/caddy-supervisor/servertype"
```

Comment:
You can plug them separate. They're independent.
I'm going to register they as 2 separate plugins on caddy developers portal.

Reply:
Okay -- but if they are in the same repo, versioning them separately will be impossible, since the build server uses `git checkout` to get the right version. Just be aware of that. :)